### PR TITLE
AWX 19.2.1

### PR DESCRIPTION
**Pull request by shanemcd**

### CODE REVIEW

**Comment by shanemcd** [comment]
recheck

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
 # Changelog
 
-This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
+# 19.2.1 (June 14, 2021)
+
+- There are now 2 default Instance Groups: 'controlplane' and 'default' (https://github.com/ansible/awx/pull/10324)
+- Removed deprecated modules: `tower_send`, `tower_receive`, `tower_workflow_template` (https://github.com/ansible/awx/pull/9980)
+- Improved UI performance when a large amount of events are being emmitted by jobs (https://github.com/ansible/awx/pull/10053)
```
Comment (jbradberry) [inline CHANGELOG.md]: emitted

Comment (shanemcd) [inline CHANGELOG.md]:
```suggestion
- Improved UI performance when a large amount of events are being emitted by jobs (https://github.com/ansible/awx/pull/10053)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+- Added favicon (https://github.com/ansible/awx/pull/10388)
+- Removed `not` option from smart inventory host filter search as it's not supported by the API (https://github.com/ansible/awx/pull/10380)
+- Added button to allow user to refetch project revision after project sync has finished (https://github.com/ansible/awx/pull/10334)
+- Fixed bug where extraneous CONFIG requests were made on logout (https://github.com/ansible/awx/pull/10379)
+- Fixed bug where users were unable to cancel inventory syncs (https://github.com/ansible/awx/pull/10346)
+- Added missing dashboard graph filters (https://github.com/ansible/awx/pull/10349)
+- Added support for typing in to single select lookup form fields (https://github.com/ansible/awx/pull/10257)
+- Fixed various bugs related to user sessions (https://github.com/ansible/awx/pull/9908)
+- Fixed bug where sorting in modals would close the modal (https://github.com/ansible/awx/pull/10215)
+- Added support for Red Hat Insights as an inventory source (https://github.com/ansible/awx/pull/8650)
```
Comment (jakemcdermott) [inline CHANGELOG.md:20]:
```suggestion
- Added support for Red Hat Insights as an inventory source (https://github.com/ansible/awx/pull/8650)
- Fixed bugs when selecting items in a list then sorting/paginating (https://github.com/ansible/awx/pull/10329)
```

**Review by jakemcdermott — APPROVED** [review]

**Review by jbradberry — APPROVED** [review]

**Review by beeankha — APPROVED** [review]

**Comment by shanemcd** [comment]
regate

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,24 @@
 # Changelog
 
-This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
+# 19.2.1 (June 14, 2021)
+
+- There are now 2 default Instance Groups: 'controlplane' and 'default' (https://github.com/ansible/awx/pull/10324)
+- Removed deprecated modules: `tower_send`, `tower_receive`, `tower_workflow_template` (https://github.com/ansible/awx/pull/9980)
```
Comment (fosterseth) [inline CHANGELOG.md:6]: should we also add that `tower_` prefix has been removed from modules,  e.g. `tower_job_template` to `job_template`

**Pull request merged**